com.unity.ml-agents/Runtime/Actuators/ActionSpaceDef.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.MLAgents.Policies;
+
+namespace Unity.MLAgents.Actuators
+{
+    /// <summary>
+    /// Defines the structure of an Action Space to be used by the Actuator system.
+    /// </summary>
+    internal readonly struct ActionSpaceDef
```

Comment:
Regarding naming, we could call this `ActionSpecs` since it is similar to BehaviorSpecs.

Reply:
> I did not see the `BufferedDiscreteActionMask` mentioned in the PR comment, is it still in the PR?
> I made some initial comments but I need to review it some more.

Whoops, renamed it. I'll update the description.